src/main/scala/io/iohk/ethereum/db/components/StoragesComp.scala

```diff
@@ -0,0 +1,25 @@
+package io.iohk.ethereum.db.components
+
+import io.iohk.ethereum.db.storage._
+
+trait StoragesComp {
+
+  def storages: Storages
+
+  trait Storages {
+
+    def blockHeadersStorage: BlockHeadersStorage
```

Comment:
Well...that's the point. I wanted to leave this open in order not to be too strict regarding if we are using a single instance. But as you suggest this might lead to confusion, I can change it to `vals` and then refactor this in case we no longer need a single reference, how does it sound?